dart_assignment1/lib/component/Item.dart

```diff
@@ -1,46 +1,30 @@
-class Item
+import 'dart:core';
```

Comment:
this file is not formatted well use proper formatting shortcuts for formatting. `Shift + Option + F ` should work for vscode.